source/features/first-published-tag-for-merged-pr.tsx

```diff
@@ -77,14 +77,17 @@ function addExistingTagLink(tagName: string): void {
 }
 
 function addLinkToCreateRelease(text = 'Now you can release this change'): void {
+	const url = isRefinedGitHubRepo()
+		? 'https://github.com/refined-github/refined-github/actions/workflows/release.yml'
```

Comment:
Neat! Now if we have something like `buildRefinedGitHubRepoURL`...

Reply:
We do have `getRghIssueUrl` 😃 but no generic version since they're otherwise rare